qtpy/QtWebEngineWidgets.py

```diff
@@ -10,7 +10,7 @@
 Provides QtWebEngineWidgets classes and functions.
 """
 
-from . import PYQT5,PYSIDE2, PYQT4, PYSIDE, PythonQtError
+from . import PYQT5,PYSIDE2, PYSIDE6, PYQT4, PYSIDE, PythonQtError
```

Comment:
Seems like the PYSIDE6 constant is not being used here. Is there some missing validation to do here or maybe is not necessary?